streaming-akka/src/main/scala/org/apache/spark/streaming/akka/ActorReceiver.scala

```diff
@@ -65,6 +65,7 @@ object ActorReceiver {
     val akkaConf = ConfigFactory.parseString(
       s"""akka.actor.provider = "akka.remote.RemoteActorRefProvider"
          |akka.remote.enabled-transports = ["akka.remote.netty.tcp"]
+         |akka.remote.netty.tcp.port = "0"
```

Comment:
we set it to `"0"` to have the port chosen automatically for the _Feeder_ actor and the _Receiver_ actor will "pick it up" when it subscribes to the _Feeder_ actor [Akka Remoting docs](http://doc.akka.io/docs/akka/2.3.11/scala/remoting.html#Preparing_your_ActorSystem_for_Remoting)